controllers/admin/AdminCarrierWizardController.php

```diff
@@ -43,7 +43,7 @@ public function __construct()
         $this->step_number = 0;
         $this->type_context = Shop::getContext();
         $this->old_context = Context::getContext();
-        $this->multishop_context = Shop::CONTEXT_ALL;
+        
```

Comment:
Watch out, you have some blank space in your line breake. Please remove so travis can pass.

Reply:
Or you can run cs-fixer to fix it